### PR TITLE
Added support for PlaceholderAPI 1.4^

### PR DESCRIPTION
**Pull request by MEFRREEX**

### CODE REVIEW

File: src/main/java/me/petterim1/welcomeform/Main.java [hunk 1]
```diff
@@ -96,24 +96,20 @@ public void onFormResponse(PlayerFormRespondedEvent e) {
...
-            placeholderFunc = (s, p) -> {
-                try {
-                    return (String) placeholderAPI.getDeclaredMethod("translateString", String.class, Player.class).invoke(PlaceholderAPI.getInstance(), s, p);
-                } catch (Exception e) {
-                    getLogger().error("Error with PlaceholderAPI", e);
-                    return s;
-                }
-            };
-        } catch (ClassNotFoundException ignored) {
-            placeholderFunc = (s, p) -> s;
```
Comment (PetteriM1) [inline src/main/java/me/petterim1/welcomeform/Main.java:116]: Please allow the plugin to work without PlaceholderAPI

Comment (MEFRREEX) [reply src/main/java/me/petterim1/welcomeform/Main.java:116]: fixed

File: src/main/java/me/petterim1/welcomeform/Main.java [hunk 2]
```diff
@@ -77,39 +77,44 @@ public void run() {
...
+        FormWindowSimple form = new FormWindowSimple(placeholderFunc.apply(formTitle, p), placeholderFunc.apply(formText, p));
+        if (showButton) form.addButton(new ElementButton(placeholderFunc.apply(buttonText, p)));
+        p.showFormWindow(form);
     }
 
     private void initPlaceholders() {
         try {
-            api = PlaceholderAPI.getInstance();
-        } catch (Throwable ignored) {
+            Class<?> placeholderAPI = Class.forName("com.creeperface.nukkit.placeholderapi.api.PlaceholderAPI");
```
Comment (PetteriM1) [inline src/main/java/me/petterim1/welcomeform/Main.java]: If not using reflection could we do this only once on plugin startup? Or rather check if the PlaceholderAPI plugin is loaded

Comment (MEFRREEX) [reply src/main/java/me/petterim1/welcomeform/Main.java]: Maybe so?

**Comment by PetteriM1** [comment]
Fixed in 9907d65

**Pull request closed without merging**